ci-operator/jobs/openshift/openshift-ansible/openshift-openshift-ansible-presubmits.yaml

```diff
@@ -308,6 +308,197 @@ presubmits:
             --template=/usr/local/e2e-gcp-crio \
             --target=e2e-gcp-crio
 
+  - name: pull-ci-openshift-openshift-ansible-e2e-gcp-upgrade-310-to-310x
+    agent: kubernetes
+    context: ci/prow/gcp-minor-upgrade
+    branches:
+    - release-3.10
+    rerun_command: "/test gcp-minor-upgrade"
+    always_run: true
+    trigger: "((?m)^/test( all| gcp-minor-upgrade),?(\\s+|$))"
+    decorate: true
+    skip_cloning: true
+    spec:
+      serviceAccountName: ci-operator
+      volumes:
+      - name: job-definition
+        configMap:
+          name: prow-job-cluster-launch-e2e-upgrade
+      - name: cluster-profile
+        projected:
+          sources:
+          - secret:
+              name: cluster-secrets-gcp
+          - configMap:
+              name: cluster-profile-gcp
+      containers:
+      - name: test
+        image: ci-operator:latest
+        resources: {"requests": {"cpu": "10m"},"limits":{"cpu":"500m"}}
+        volumeMounts:
+        - name: job-definition
+          mountPath: /usr/local/e2e-gcp
+          subPath: cluster-launch-e2e-upgrade.yaml
+        - name: cluster-profile
+          mountPath: /usr/local/e2e-gcp-cluster-profile
+        env:
+        - name: TEST_FOCUS
+          value: Suite:openshift/conformance/parallel
+        - name: JOB_NAME_SAFE
+          value: e2e-gcp
+        - name: CLUSTER_TYPE
+          value: gcp
+        - name: TEST_UPGRADE
+          value: "v3_10"
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              name: ci-operator-openshift-openshift-ansible
+              key: release-3.10.json
+        - name: RPM_REPO_BASEURL_REF
+          value: https://storage.googleapis.com/origin-ci-test/releases/openshift/origin/release-3.10/.latest-rpms
+        command:
+        - /bin/bash
+        - -c
+        - |
+          #!/bin/bash
+          set -e
+          export RPM_REPO="$( curl -q "${RPM_REPO_BASEURL_REF}" 2>/dev/null)"
+          export IMAGE_UPGRADE_ANSIBLE=${IMAGE_ANSIBLE}
+          export UPGRADE_RPM_REPO=${RPM_REPO}
+          ci-operator \
+            --artifact-dir=$(ARTIFACTS) \
+            --secret-dir=/usr/local/e2e-gcp-cluster-profile --template=/usr/local/e2e-gcp \
+            --target=e2e-gcp
```

Comment:
These template names need to be unique for each job that can run on a PR

Reply:
So its not tied to context or profile - and just have to be unique? Okay, got it